man/fi_av.3.md

```diff
@@ -197,6 +199,10 @@ struct fi_av_attr {
   of the first address inserted into an FI_AV_TABLE will be 0, and
   successive insertions will be given sequential indices.  Sequential
   indices will be assigned across insertion calls on the same AV.
+  Because the fi_addr_t values returned from an insertion call are
+  deterministic, applications may not need to provide the fi_addr_t
+  output parameters to insertion calls.  The exception is when
+  authentication keys are required for communication.
```

Comment:
The removed part just enumerates a few possible cases which table index as one of them. There are removed because we don't really need those details here, especially since we are deprecating the av map semantics.

The upper 32-bit usage is in a later patch, we don't need to count for that now.

Reply:
I thought another exception is `FI_AV_TABLE_USER_ID`? In that case `fi_addr_t` is also an input.

Reply:
I am changing this to: ` The exception is when the fi_addr_t parameters are also used as input for supplying    authentication keys or user defined IDs`